Client/src/python/dbs/apis/dbsClient.py

```diff
@@ -317,6 +317,22 @@ def insertBulkBlock(self, blockDump):
         :type blockDump: dict
 
         """
+        isEventCount = True
+        for f in blockDump['files']:
+            if not isEventCount: break
+            for fl in f['file_lumi_list']:
+                if 'event_count' in fl:
+                    if fl['event_count'] is None:
+                        isEventCount = False
+                        break
+                else:
+                    isEventCount = False
+                    break
```

Comment:
Can we do some thing like this. (Haven't tested yet)
```
noneEventCount = (fl.get('event_count') == None for f in blockDump['files'] for fl in f['file_lumi_list'])
noEventsFlag = any(noneEventCount)

```
and noEventsFlag == not isEventCount